spec/phrase/ota/backend_spec.rb

```diff
@@ -71,6 +72,29 @@
           expect(I18n.t(:lorem, locale: :en)).to eq("Hello OTA New")
         end
       end
+
+      context "store current_version per locale" do
+        let(:available_locales) { [:en, :fr] }
+        let(:body_fr) { {"fr" => {"lorem" => "Bonjour le monde"}}.to_yaml }
+        let(:url_fr) { "https://ota.eu.phrase.com/#{distribution_id}/#{secret_token}/fr/yml?app_version&client=ruby&sdk_version=#{Phrase::Ota::VERSION}" }
+        let(:url_en_2) { "https://ota.eu.phrase.com/#{distribution_id}/#{secret_token}/en/yml?app_version&client=ruby&current_version=0&sdk_version=#{Phrase::Ota::VERSION}" }
+
+        before do
+          stub_request(:get, url_en).to_return(status: 200, body: body_en)
+          stub_request(:get, url_en_2).to_return(status: 200, body: body_en)
+          stub_request(:get, url_fr).to_return({status: 404}, {status: 200, body: body_fr})
+        end
+
+        it do
+          subject.send(:update_translations)
+          expect(I18n.t(:lorem, locale: :en)).to eq("Hello OTA")
+          expect(I18n.t(:lorem, locale: :fr)).to eq("Translation missing: fr.lorem")
+
+          subject.send(:update_translations)
+          expect(I18n.t(:lorem, locale: :en)).to eq("Hello OTA")
+          expect(I18n.t(:lorem, locale: :fr)).to eq("Bonjour le monde")
+        end
```

Comment:
Indirectly this is testing the cache key is set. The second en request contain a value for `current_version` and `fr` does not because there has no version been fetched yet. I also kinda would like to make this more explicit but the cache version is not exposed in any other way from the backend and I don't want to test internal methods

Reply:
Makes sense.